sale_comment_template/views/base_comment_template_view.xml

```diff
@@ -5,11 +5,8 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">base.comment.template</field>
         <field name="view_mode">tree,form</field>
-        <field name="domain">[('model_ids.model', '=', 'sale.order')]</field>
-        <field
-            name="context"
-            eval="{'default_model_ids': [(4, ref('sale.model_sale_order'))]}"
-        />
+        <field name="domain">[('models', 'like', 'sale.order')]</field>
```

Comment:
This is not correct, as if you have a template for `sale.order.line`, it will be shown.

Reply:
But it will fail if you share this template with account.move
This template couldn't be shared with another model

What do you propose? @pedrobaeza

Reply:
Use the proper domain. There is another m2m field if I don't remember bad for filtering directly against `ir.model`, and using `ref`, you can get what is needed.

Reply:
Thanks for your help

I have tested with 

```xml
<field name="domain" eval="[('model_ids.id', '=', ref('sale.model_sale_order'))]" />
```
```xml
<field name="domain" eval="[('model_ids, '=', ref('sale.model_sale_order'))]" />
```
```xml
<field name="domain">[('model_ids.model', '=', 'sale.order')]</field>
```

None of this works, the action returns an empty page

Any idea how can I implement this domain correctly?

Reply:
Another option is to not filter at all... 

https://github.com/OCA/account-invoice-reporting/blob/16.0/account_comment_template/views/base_comment_template_view.xml

Only show the menuitem without any domain

Reply:
Search method seems to not be well implemented:

https://github.com/OCA/reporting-engine/blob/16.0/base_comment_template/models/base_comment_template.py#L119